index.html

```diff
@@ -20,10 +20,70 @@
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
-    <!--
+    <header>
+      <ul class="header">
+        <h1 class="logo">Fitzone</h1>
+        <h2 class="nav1">HOME</h2>
+        <h2 class="nav2">ABOUT US</h2>
+        <h2 class="nav3">MEMBERSHIP</h2>
+        <h2 class="nav4">LOCATIONS</h2>
+        <h2 class="signup">SIGN UP</h2>
+        <h2 class="login">LOGIN</h2>
+      </ul>
+    </header>
+    <main>
+      <div class="banner">
+        <p class="p1">
+          Fitzone is commited to providing the best possible<br> 
```

Comment:
The `main` element needs to have an `h1`, which is missing currently.

Reply:
If your design doesn't permit it, you can hide the headings so that they are read out by screen readers but not seen by visual users. 

Here is an example of some CSS to implement a screen reader only text class. 

https://tailwindcss.com/docs/screen-readers